cmd/kk/pkg/kubernetes/tasks.go

```diff
@@ -154,11 +154,11 @@ func SyncKubeBinaries(runtime connector.Runtime, binariesMap map[string]*files.K
 	return nil
 }
 
-type SyncKubelet struct {
+type ChownKubelet struct {
 	common.KubeAction
 }
 
-func (s *SyncKubelet) Execute(runtime connector.Runtime) error {
+func (c *ChownKubelet) Execute(runtime connector.Runtime) error {
```

Comment:
`chown` may need to be replaced with `chmod`.

Reply:
Ok, I'll re-push the pr after modifying the content.

Reply:
> `chown` may need to be replaced with `chmod`.

Ok, I'll re-push the pr after modifying the content.